### DISCUSSION TITLE
Is there a Discord Server?

### DISCUSSION THREAD

**Discussion by Skillz4Killz** [prisma/prisma#3920] [Q&A] [answered]
Title says it all. Is there any official discord server for Prisma instead of Slack?

**Comment by ryands17** [comment]
Currently no there isn't any and keeping two separate places for support would be inefficient anyway. So it's unlikely that there ever would be a discord server along with Slack.

**Reply by Skillz4Killz** [reply]
Is it worth discussing a move from slack to discord if the goal is to keep 1 place?

**Reply by ryands17** [reply]
Any specific reason to move to Discord from Slack?

**Reply by Skillz4Killz** [reply]
Almost all the open-source tools I use have already migrated to Discord. https://discord.com/open-source Having to reinstall slack just for Prisma isn't exactly something I would want to do. It just becomes a barrier for support.

**Reply by pantharshit00** [reply]
@Skillz4Killz We have a discord server for our Rust crates: https://discord.gg/6FrFhYA

It is not very active but it exists. 

We have over 40,000 members on slack now so moving all of them to discord isn't really viable for us now.

**Reply by Skillz4Killz** [reply]
I think the activity and size of that Discord would grow if it was placed on the community page. Anyway that Discord is great start!

**Reply by leonardssh** [reply]
Isn't there a discord server yet? 🤔

**Reply by pantharshit00** [reply]
@LeonardSSH We have a slack: https://slack.prisma.io

**Reply by Olyno** [reply]
Well, in my opinion you're wrong to think like this. Yes you have a lot of members on Slack, but **a lot** of new technologies (just to give some Yarn, Typescript, Deno, Svelte, Nestjs) have a discord for the next reasons:
 - Easy to setup
 - Easy to manage
 - A bigger community (a lot of discord users are developers)
 - Faster to share/start
 - Better design (sorry for slack but it's a fact)
 - A lot of tools of Discord are open sources

Prisma's place should not be on Slack but on Discord because it's simply a better place to make it growth. If you need a begin template, you can use one of mine [by clicking here](https://discord.new/HadbXtHNzvrH).

Again, it's only my personal opinion about this topic, and won't to force anyone to move from Slack, but I think a lot of members of the community will be agree with this change.

**Reply by eliorivero** [reply]
I'll add a vote for Discord. I definitely can't explain why, I just know that I participate more in Discord communities than Slack ones. I mean, there's a gotta be a reason why Next.js, Go, Rust and others have Discord communities. Maybe Slack feels like work? The environment often sets the mood, and IMHO the environment in Discord is more relaxed than Slack. You could just do an experiment: add a Discord link next to the Slack link on the site. After a couple of weeks or a month, check which community grew more.

**Reply by johnsonthedev** [reply]
if Prisma won't do it I am wondering are we allowed to create an unofficial Prisma channel on discord?

**Reply by Skillz4Killz** [reply]
I would be interested in one for sure!

**Reply by currenthandle** [reply]
Bump. Still no Discord?

**Reply by leonardssh** [reply]
@caseysiebel nope, still waiting...

**Reply by pgiovanni** [reply]
we want a discord :)

**Reply by leonardssh** [reply]
@pgiovanni yeap, still waiting... 😅

**Reply by Olyno** [reply]
Because many members have been asking for it for a while, I have dedicated myself to creating a [community discord for Prisma](https://discord.gg/Wa7fU7dqqS). I would be happy to welcome anyone, including maintainers of Prisma, to the discord.

**Note:** This is an unofficial discord server. Please refer to Github or Slack to have official team members and help.

**Reply by HarryET** [reply]
> Because many members have been asking for it for a while, I have dedicated myself to creating a [community discord for Prisma](https://discord.gg/Wa7fU7dqqS). I would be happy to welcome anyone, including maintainers of Prisma, to the discord.

Awesome to see! The discord link does app ear to be broken though as there is an extra dot at the end of the invite string 😓

**Reply by Olyno** [reply]
Fixed, thanks you for the report!

**Reply by vladi-stevanovic** [reply]
Hello everyone, 
Let me start by saying that we appreciate your enthusiasm to see the Prisma community grow on Discord and we hear your feedback! 

It is on our roadmap to evaluate opening an official Discord channel in the coming months, however, we don’t have the people power to moderate and support one currently.

@Olyno has kindly created an **unofficial Prisma Discord channel**, and they’ll be providing moderation, to ensure a positive, inclusive environment focused on learning and growth. 

Please note that this Discord will **not** be staffed by Prisma at present. For support by our team, you will need to reach out on our [GitHub Discussions](https://github.com/prisma/prisma/discussions) or on our [Slack](https://slack.prisma.io/). 

Thank you again for your enthusiasm and feedback, and we’ll keep you posted as we grow our Prisma Dev Success Team and the channels we can support.

**Reply by ImLunaHey** [reply]
Are there any changes on this?

**Reply by nikolasburk** [reply]
Hey @ImLunaHey, there are no changes at the moment. Here are the best places to connect with the Prisma community:

- Official
  - [GitHub](https://github.com/prisma/prisma/) (issues & discussions)
  - [Slack](https://slack.prisma.io) (help, discussions, showcases, job board, feedback, ...)
- Unofficial
  - [Discord](https://discord.com/invite/Wa7fU7dqqS) (community conversations)

**Reply by jharrell** [reply]
Just wanted to chime in on this thread that we've adopted the community discord and it's now official. We're still in the process of updating all of the things, but yeah! It's happening. https://github.com/prisma/prisma/discussions/3920#discussioncomment-6199579

Blog post: https://www.prisma.io/blog/announcing-discord-1LiAOpS7lxV9
Invite link: [discord.gg/jS3XY7vp46](https://discord.gg/jS3XY7vp46)

**Comment by jdgamble555** [comment]
Hey guys, I though I would add here that I say the Fireship move from Slack to Discord grow from 19k to 30k within 6 months of the move. There are more active users on Discord, and it is cheaper. From a business perspective, it is cheaper and you get more user interactions. Just FWI.

J

**Comment by jharrell** [comment, marked as answer]
I'm super excited to announce... 🥁 🥁 🥁 

We now have an [official Discord!](https://discord.gg/jS3XY7vp46)

Thank you so much to the efforts of @Olyno for creating and maintaining an active community discord. We're extremely excited to officially adopt their Discord as our official Discord server. 

What this means for the community is simple: you now have more ways to reach and interact with the Prisma team. We will continue to be active in our existing official forums (GitHub and Slack) as well as Discord. 

As always, I will mention that for official discussions, bug reports, or feature requests we recommend Github issues and discussions. 

Join now! https://discord.gg/jS3XY7vp46

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
